shared/tasks/browser.mjs

```diff
@@ -56,6 +68,31 @@ export async function screenshots () {
     await Promise.all(screenshotTasks)
   }
 
+  // Extra 'manual' set of screenshots for rebrand examples
+  for (const rebrandExample of rebrandExamples) {
+    let page = await browser.newPage()
+    const cookie = {
```

Comment:
I think this cookie will persist in the browser environment between screenshots. Not a huge issue at the minute as we're doing these last, but I wonder if it'd be simpler and safer to use the `rebrandOverride` url parameter instead?